components/bl-list-component/src/list-content.js

```diff
@@ -0,0 +1,15 @@
+import { List } from "./list";
+
+export function ListContent({ props, items }) {
+  return (items || []).map((item, i) => (
+    <li
+      key={i}
+      className="list__item"
+      style={{ color: props.color, fontSize: props.fontSize }}
+      onClick={(e) => props.onClickHandler(e, item)}
```

Comment:
Props like `color/fontSize/onItemClick/list type` is stable and everywhere the same, use a react context and pass all of them through it